.gitignore

```diff
@@ -34,8 +34,8 @@ yarn-error.log*
 # typescript
 *.tsbuildinfo
 
-# GraphQL auto-generated
-generated
```

Comment:
@timuric let's exclude `generated/graphql.ts` (TypeScript) so it's consistent with what we agreed on, i.e. Schema SDL committed in users repo `generated/schema.graphql`, TypeScript types generated on the fly per `dev` and `builld`

Reply:
done

Reply:
@timuric thanks